pwa/src/components/organizationCard/OrganizationCard.module.css

```diff
@@ -0,0 +1,68 @@
+.container {
+  border: var(--web-app-component-card-border-color);
+  padding-inline-start: var(--web-app-size-md);
+  padding-inline-end: var(--web-app-size-md);
+  padding-block-start: var(--web-app-size-md);
+  padding-block-end: var(--web-app-size-md);
+}
+
+.container > *:not(:last-child) {
+  margin-block-end: var(--web-app-size-md);
+}
+
+.header {
+  display: flex;
+}
+
+.header > *:not(:last-child) {
+  margin-inline-end: var(--web-app-size-md);
+}
+
+.header > .headerContent {
+  flex: 1;
+}
+
+.titleLink {
+  font-size: var(--web-app-font-size-lg);
+  --denhaag-link-icon-gap: var(--web-app-size-xs);
```

Comment:
I don't think we're actually directly assigning values to any "outside" tokens, except for in `component-overrides.css`. 

With that in mind, I'm thinking this might better be solved differently (e.g. by making a `--web-app` token that is used to override the `--denhaag` token _within_ the `component-overrides.css`).